compute/admin_guide/vulnerability_management/registry_scanning/nexus-registry.adoc

```diff
@@ -0,0 +1,116 @@
+== Sonatype Nexus Registry
+
+Scan Nexus registries.
+
+To scan a repository in Sonatype Nexus Registry, create a new registry scan setting.
+
+[#nexus-repo-connector]
+==== Prerequisites
+
+* You have xref:../../install/defender_types.adoc#[installed a Defender] somewhere in your environment. 
+
+* The Defender can establish a connection with the Nexus registry over port 443.
++
+Ensure that the port is open for the image to be accessed successfully.
+
+* Configure the connector port for the Docker engine to connect to the Nexus registry.
```

Comment:
Is this a Nexus requirement? Can you add a sentence to say why this connector port is needed?